pandas/io/pickle.py

```diff
@@ -86,13 +107,14 @@ def read_pickle(path, compression="infer"):
 
     Parameters
     ----------
-    path : str
-        File path where the pickled object will be loaded.
+    filepath_or_buffer : str, path object or file-like object
+        File path, URL, or buffer where the pickled object will be loaded from.
+        URL is not limited to S3 and GCS.
```

Comment:
versionchanged tag

Reply:
Added versionchanged tag for ver 1.0.0 to mention it accepts URLs

> ... versionchanged:: 1.0.0
> Accept URL. URL is not limited to S3 and GCS.